src/routers/semester.ts

```diff
@@ -1,21 +1,27 @@
 import express from 'express';
-import { create, read, update, del } from '../operators/semesterOperations';
+import Ajv from 'ajv';
 import Semester from '../models/semester';
+import schema from '../types/schema.json';
 
 const semesterRouter = express.Router();
+const ajv = new Ajv({ removeAdditional: true });
 
 const ERROR_RESPONSE = 'Semester not found.';
 
 semesterRouter.get('/:id', async (req, res) => {
 	try {
+		const id = req.params.id;
 		const userID = req.header('userID');
-		if (userID) {
-			const semester = await read(req.params.id, userID);
-			const resData = {
-				...semester,
-				courses: Array.from(semester?.courses || []),
-			};
-			res.send(resData);
+		if (id && userID) {
+			/* leaving this here as a reminder to add courses array if necessary */
+			// const semester = await read(req.params.id, userID);
+			// const resData = {
+			// 	...semester,
+			// 	courses: Array.from(semester?.courses || []),
+			// };
```

Comment:
This is a good comment. An aside, you might not needs the `Array.from` with how the new database stuff works